hips/draw/simple.py

```diff
@@ -61,7 +61,7 @@ class HipsPainter:
     (1000, 2000)
     """
 
-    def __init__(self, geometry: WCSGeometry, hips_survey: HipsSurveyProperties, tile_format: str) -> None:
+    def __init__(self, geometry: WCSGeometry, hips_survey: str or HipsSurveyProperties, tile_format: str) -> None:
```

Comment:
I think the type annotation has to be `Union` in case of multiple options, no?
https://docs.python.org/3/library/typing.html#typing.Union